File: src/main/java/com/github/simiacryptus/aicoder/openai/Engine.java
package com.github.simiacryptus.aicoder.openai;

public class Engine {
    @SuppressWarnings("unused")
    public String id;
    @SuppressWarnings("unused")
    public boolean ready;
    @SuppressWarnings("unused")
    public String owner;
    @SuppressWarnings("unused")
    public String object;
    @SuppressWarnings("unused")
    public Integer created;
    @SuppressWarnings("unused")
    public String permissions;
    @SuppressWarnings("unused")
    public Integer replicas;
    @SuppressWarnings("unused")
    public Integer max_replicas;

    @SuppressWarnings("unused")
    public Engine() {
    }

    @SuppressWarnings("unused")
    public Engine(String id, boolean ready, String owner, String object, Integer created, String permissions, Integer replicas, Integer max_replicas) {
        this.id = id;
        this.ready = ready;
        this.owner = owner;
        this.object = object;
        this.created = created;
        this.permissions = permissions;
        this.replicas = replicas;
        this.max_replicas = max_replicas;
    }

}

File: src/main/kotlin/com/github/simiacryptus/openai/Engine.kt
package com.github.simiacryptus.openai

class Engine {
    @Suppress("unused")
    var id: String? = null

    @Suppress("unused")
    var ready = false

    @Suppress("unused")
    var owner: String? = null

    @Suppress("unused")
    var `object`: String? = null

    @Suppress("unused")
    var created: Int? = null

    @Suppress("unused")
    var permissions: String? = null

    @Suppress("unused")
    var replicas: Int? = null

    @Suppress("unused")
    var max_replicas: Int? = null

    @Suppress("unused")
    constructor()

    @Suppress("unused")
    constructor(
        id: String?,
        ready: Boolean,
        owner: String?,
        `object`: String?,
        created: Int?,
        permissions: String?,
        replicas: Int?,
        max_replicas: Int?
    ) {
        this.id = id
        this.ready = ready
        this.owner = owner
        this.`object` = `object`
        this.created = created
        this.permissions = permissions
        this.replicas = replicas
        this.max_replicas = max_replicas
    }
}